tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "target": "ES2018",
     "module": "commonjs",
     "lib": ["es2018"],
-    "declaration": true,
+    "declaration": false,
+    "noEmit": true,
```

Comment:
Why are we enabling noEmit?

Reply:
The idea is we don't need compiler output for the CDK constructs.